chainerx_cc/chainerx/routines/creation.cc

```diff
@@ -345,4 +347,60 @@ Array Linspace(
     return out;
 }
 
+std::vector<Array> Meshgrid(const std::vector<Array>& arrays, MeshgridIndexingMode mode) {
+    Shape shape;
+    Shape broadcast_shape;
+    std::vector<Shape> arr_reshapes;
```

Comment:
`arr_reshapes` can also be `reserve`d? And a detail but how about keeping the names somewhat aligned `s/arr/array`.

Reply:
reserve done.
> And a detail but how about keeping the names somewhat aligned s/arr/array.

Didn't understand this part exactly.
Have renamed `arr_reshapes` to `broadcasted_array_shapes`